.cirrus.yml

```diff
@@ -16,6 +16,10 @@ fedora_30_task:
     matrix:
       - SETUP: setup.py develop --user
       - SETUP: -m pip install .
+  # Latest pip available avocado-71.0 contains a bug
```

Comment:
Which bug?

Reply:
This one https://github.com/avocado-framework/avocado/pull/3309